usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -114,6 +119,15 @@ def subaward_annotations():
     annotation_fields = {
         "subaward_action_date_fiscal_year": FiscalYear("subaward__action_date"),
         "prime_award_base_action_date_fiscal_year": FiscalYear("award__date_signed"),
+        "prime_award_period_of_performance_start_date": Cast(
```

Comment:
Why do these need to be cast if they are already `DateField` on the Award model?

Reply:
They don't! For some reason i thought the ticket said DateTimeField and not DateField